src/Context.ts

```diff
@@ -279,20 +279,20 @@ export class Context {
       );
     }
   }
-  _applyLineCap(shape) {
-    var lineCap = shape.getLineCap();
+  _applyLineCap(shape: Shape) {
+    const lineCap = shape.lineCap();
```

Comment:
`shape.getLineCap()` is a bit faster than `shape.lineCap()` so I would prefer keep using it instead. Not sure what to do with types here, because I don't want to define all getters and setters.